### ISSUE TITLE
Test common navigation options for screen reader users

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1794]
As a screen reader user, I expect the common navigation tools I use to be helpful for providing me with the necessary information to get an overview of the page and quickly navigate to content that meets my information need.

Using NVDA, test the three most common navigation options screen reader users utilize when reading a page. For each page, list out 1) buttons, 2) links, and 3) headings. Document what is presented to users and create new tickets where navigation information is insufficient, misleading, or confusing.

- [ ] Fulcrum homepage
- [ ] Publisher page
- [ ] Monograph catalog page
- [ ] Fulcrum E-Reader
- [ ] Video asset
- [ ] Audio asset
- [ ] Image asset
- [ ] Document asset
